Ruby Basic/task6.rb

```diff
@@ -0,0 +1,38 @@
+class Numbers
+  def initialize(first_number, second_number)
+    @first_number = first_number.to_i
+    @second_number = second_number.to_i
+  end
+
+  def sum
+    @first_number + @second_number
+  end
+
+  def subtraction
+    @first_number - @second_number
+  end
+
+  def multiplication
+    @first_number * @second_number
+  end
+
+  def division
+    @first_number / @second_number
+  end
+end
+
+input_data = ARGV
```

Comment:
`input_data` is useless variable, you can use `ARGV `

Reply:
Done